types/contract-information.type.ts

```diff
@@ -0,0 +1,20 @@
+export type ContractInformation = {
```

Comment:
Most, if not all of the types here (and maybe in the ChainInformation type) should be optional, right?

I think in any case, when we pull this in for use with glacier we will be treating them all as optional.

Reply:
There are two reasons to make data required:

1. There is a technical need for it (ex: address).
2. The data is easy to fill out so we should require it for the sake of enforcing that users actually fill out the data (ex: contractName).  More data is better for the sake of filling out relevant content in the UI.

Based on this, IMO for contract information the only **optional** values should be:

1. assetType
2. officialSite
3. tags
4. logoUri
5. resourceLinks
6. symbol

Happy to discuss more if needed.

Reply:
Agreed that the optional values should also be marked as optional in this type file too.